konf/site.yaml

```diff
@@ -2,7 +2,9 @@ domain: jp.ubuntu.com
 
 image: prod-comms.docker-registry.canonical.com/jp.ubuntu.com
 
-useProxy: false
+useProxy: true
```

Comment:
Why?

Reply:
Well, I'd kind of expect it to either work or not. It would be strange for it to be causing *intermittent* 503s. I don't think the blog should go through the proxy, actually, because admin.insights.ubuntu.com is one of our domains so we don't normally use the proxy for it. It's only available over the VPN, not externally anyway.

But it should also be explicitly excluded from going through the proxy anyway ([see here](https://github.com/canonical/konf/blob/00e89099ef8361e26df2e0f336c8651d333bb9fa/templates/proxy-settings.yaml)). So in theory it doesn't really matter which you do, `true` or `false`, unless there's something else that jp.ubuntu.com needs access to. The search API maybe?

If there isn't anything then it's probably best to just delete this rule and let it fall back to whatever the default is (it looks to me like [the default `true`](https://github.com/canonical/konf/blob/00e89099ef8361e26df2e0f336c8651d333bb9fa/templates/deployment.yaml#L51)).

Reply:
Ok. I kept it at `false`